frontend/build/profiles/integration/annotation-tool-configuration.js

```diff
@@ -151,20 +153,194 @@ define(["jquery",
              * @param {Category[]} categories The tracks to include in the export
              * @param {Boolean} freeText Should free-text annotations be exported?
              */
-            export: function (video, tracks, categories, freeText) {
-                var parameters = new URLSearchParams();
+            export_csv: function (video, tracks, categories, freeText) {
```

Comment:
Function and variable names in JavaScript should be in `camelCase`.

```suggestion
            exportCSV: function (video, tracks, categories, freeText) {
```

(Note: Whether it should be `exportCsv` or `exportCSV` is debatable. I prefer the latter, because CSV is an acronym, so in a way, each letter is its own word.)

(Also note that I won't mention these kinds of style and naming issues on every instance; you are invited to look for potential further places to fix them yourself. x))

Reply:
(And I've switched to the former, as that appears more readable to me when the acronym is followed by another word. )

(These reminders are still very appreciated :bow: )